README.md

```diff
@@ -60,6 +60,16 @@ Running this command will:
 
 For convenience of using `phpcs` as global command you might want to add path to `wpcs/vendor/bin` directory to a `PATH` environment of your operating system.
 
+#### Installing WPCS as a dependency
+
+When installing the WordPress Coding Standards as a dependency in a larger project, the above mentioned step 3 will not be executed automatically.
+
+There are two actively maintained Composer plugins which can handle the registration of standards with PHP_CodeSniffer for you:
+* [composer-phpcodesniffer-standards-plugin](https://github.com/higidi/composer-phpcodesniffer-standards-plugin)
+* [phpcodesniffer-composer-installer](https://github.com/DealerDirect/phpcodesniffer-composer-installer)
```

Comment:
> Do these two both work in roughly the same way?

AFAIK yes, they do.

> How would someone choose between the two?

Probably the same way as when choosing between two different WP plugins to use:
* See if it's still actively maintained, issues are being responded to, has good reviews, how many users it has on packagist.
* Install each, test it, see if you like it and it does what you want.

> Worth WPCS picking one as the recommendation, and leaving the other as a link for an alternative, instead of presenting both equally (in this doc)?

We could, but I figured most Composer users would just `require` WPCS and see the `suggest` and follow that without reading the readme. I'd be happy to adjust it though.

The reason I choose the @DealerDirect version as the recommendation is that the team of that plugin has been very active trying to help external standards get this issue sorted, created their Composer plugin in response to an issue with another external standard I'm involved with and has been very responsive to queries related to the plugin.